.github/workflows/infrastructure.yml

```diff
@@ -0,0 +1,173 @@
+        'Product=${{ env.product }}' \
+        'Region=${{ vars.REGION }}' \
+        'Team=development' >> $GITHUB_ENV
+
+    - name: Login 🔐
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Azure defaults ✨
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          # Basic
+          az configure --defaults location=${{ vars.REGION }}
+          az configure --defaults group=rg-${{ env.product }}-${{ env.target }}-001
+
+          # AZ CLI extensions upgrade
+          az extension add --name containerapp --upgrade
+
+    - name: Resource group 🏗️
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          az group create \
+          --name rg-${{ env.product }}-${{ env.target }}-001 \
+          --tags ${{ env.tags }}
+
+    
+    - name: Container registry 📦️
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          az acr create \
+          --name cr${{ env.product }}${{ env.target }}${{ vars.REGION }}001 \
+          --sku Standard \
+          --admin-enabled true \
+          --tags ${{ env.tags }}
+
+    - name: Container app environment 🗃️
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          az containerapp env create \
+          --name cae-${{ env.product }}-${{ env.target }}-${{ vars.REGION }}-001 \
+          --tags ${{ env.tags }}
+
+    
+    - name: Container app - MDM 📄
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          az containerapp create \
+          --name ca-mdm-${{ env.product }}-${{ env.target }}-${{ vars.REGION }}-001 \
+          --environment cae-${{ env.product }}-${{ env.target }}-${{ vars.REGION }}-001 \
+          --registry-server cr${{ env.product }}${{ env.target }}${{ vars.REGION }}001.azurecr.io \
+          --registry-username $(az acr credential show -n cr${{ env.product }}${{ env.target }}${{ vars.REGION }}001 --query username | tr -d '"') \
+          --cpu 0.25 \
+          --memory 0.5Gi \
+          --min-replicas 1 \
+          --ingress internal \
+          --target-port ${{ vars.MDM_PORT }} \
+          --revision-suffix ${{ vars.REVISION_SUFFIX }} \
+          --revisions-mode multiple \
+          --transport http2 \
+          --tags ${{ env.tags }}
+
+    - name: Container app - TFS 📄
+      uses: azure/cli@v1.0.0
+      with:
+        inlineScript: |
+          az containerapp create \
+          --name ca-tfs-${{ env.product }}-${{ env.target }}-${{ vars.REGION }}-001 \
+          --environment cae-${{ env.product }}-${{ env.target }}-${{ vars.REGION }}-001 \
+          --registry-server cr${{ env.product }}${{ env.target }}${{ vars.REGION }}001.azurecr.io \
+          --registry-username $(az acr credential show -n cr${{ env.product }}${{ env.target }}${{ vars.REGION }}001 --query username | tr -d '"') \
+          --cpu 0.25 \
+          --memory 0.5Gi \
+          --min-replicas 1 \
```

Comment:
We need to add max replicas as well to avoid it from expanding too much

Reply:
Have added.